### DISCUSSION TITLE
Swheatz - Stellar Quest Submission Chapter2

### DISCUSSION THREAD

**Discussion by swheatz** [stellar/sorobanathon#120] [General]
![rwxquest](https://user-images.githubusercontent.com/120219297/206788142-ed3fda28-1d16-4572-b630-d60d282e6113.png)

Swheatz - Stellar Quest Submission Chapter2

**Comment by stellarjohn** [comment]
Hi Swheatz!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
